src/viur_cli/local.py

```diff
@@ -229,3 +233,86 @@ def show_output_if_not(args, check_str):
+    Check for npm vulnerabilities in the project and optionally fix them.
+
+    This function runs the "npm audit" command to check for vulnerabilities in the project's dependencies. If any
+    vulnerabilities are found, the function prompts the user to view the details and optionally run "npm audit fix
+    --force" to automatically fix the vulnerabilities.
+
+    Args:
+        autofix (bool): A boolean indicating whether to automatically fix vulnerabilities.
+
+    Raises:
+        subprocess.CalledProcessError: If an error occurs while running the "npm audit" or "npm audit fix" commands.
+    """
+    sources_folder = './sources'
+
+    try:
+        # Run "npm audit" command and capture the output
+        result = subprocess.run(
+            ['npm', 'audit', '--json'],
+            capture_output=True,
+            cwd=sources_folder,
+            encoding='utf-8'
+        )
+
+        vulnerabilities = json.loads(result.stdout)["metadata"]["vulnerabilities"]
+
+        if vulnerabilities["total"] >= 0:
+
+            if not autofix:
+                print(
+                    f'Npm found {vulnerabilities["total"]} Vulnerabilities \n'
+                    f'info: {vulnerabilities["info"]}\n'
+                    f'low: {vulnerabilities["low"]}\n'
+                    f'moderate: {vulnerabilities["moderate"]}\n'
+                    f'high: {vulnerabilities["high"]}\n'
+                    f'critical: {vulnerabilities["critical"]}\n'
+                    f'total: {vulnerabilities["total"]}\n'
+                )
+                show_vulnerabilities = input('Do you want a list of the found Vulnerabilities? (y/N)').strip().lower()
+
+                if show_vulnerabilities == 'y':
+                    pprint(json.loads(result.stdout)["vulnerabilities"])
+
+                confirm = input('Do you want to run "npm audit fix --force" automatically? (Y/n):').strip().lower()
+
+            if autofix or confirm == 'y' or confirm == '':
+
+                try:
+                    fix = subprocess.run(
+                        ["npm", "audit", "fix", "--force",  "--json"],
+                        capture_output=True,
+                        cwd=sources_folder,
+                        encoding="utf-8",
+                    )
+
+                    fix_output = json.loads(fix.stdout)
+
+                    print(
+                        f'npm added {fix_output["added"]} packages,\n '
+                        f'audited {fix_output["audited"]} packages,\n '
+                        f'changed {fix_output["changed"]} packages\n '
+                        f'and removed {fix_output["removed"]} packages\n'
+                    )
+
+                    show_all_fix = input('Do you want the whole report of the npm fixes?(y/N)').lower().strip()
+                    if show_all_fix == 'y':
+                        pprint(fix_output)
+
+                except Exception as e:
+                    print(f'{e}')
+            else:
+                print(
```

Comment:
Maybe `echo_info` would be more nice.
```suggestion
                echo_info(
```